sdks/java/core/src/main/java/org/apache/beam/sdk/util/ReifyTimestampsAndWindows.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2015 Google Inc.
```

Comment:
This is the wrong license. have you run `mvn verify` and/or whichever one verifies licenses and checkstyle?